scripts/subscribeEventProof.js

```diff
@@ -116,7 +65,22 @@ async function main (networkName, bridgeContractAddress) {
     logger.info(`CENNZnet bridge deployed to: ${bridge.address}`);
     logger.info(`Executor: ${txExecutor.address}`);
 
-    let lastValidatorsSet = getValidatorAddedToBridge(bridge);
+    // For any kind of restart, we check if the last event proof generated on CENNZnet side, has been update on Eth side
+    logger.info('Check the last event proof generated on CENNZnet side, has been update on Eth side');
```

Comment:
there should be some check if the CENNZnet proof is > Eth proof  + 1, it will need to be handled differently. e.g. replay all prior proofs and validator sets up until this point. easiest to log it for now as it should not be offline longer than that